internal/cli/apps.go

```diff
@@ -668,6 +674,17 @@ func apiAuthMethodFor(v string) *string {
 	}
 }
 
+func apiDefaultAuthMethodFor(t string) *string {
+	switch apiTypeFor(strings.ToLower(t)) {
+	case appTypeNative:
+		fallthrough
+	case appTypeSPA:
+		return auth0.String("none")
```

Comment:
Just so I understand, we want `none` for both native/spa, correct?

Reply:
Exactly, both SPA and native